manifests/cluster-manager/hub/cluster-manager-registration-serviceaccount.yaml

```diff
@@ -3,3 +3,7 @@ kind: ServiceAccount
 metadata:
   name: registration-controller-sa
   namespace: {{ .ClusterManagerNamespace }}
+  {{ if .ManagedClusterIdentityCreatorRole }}
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .ManagedClusterIdentityCreatorRole }}
```

Comment:
this is not quite extensible, e.g. the key is specifically for eks. How do you think the keys/values could be configured from the template?

Reply:
I think same needs to be applied on klusterlet SA as well. Will address it in a seperate PR.